source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,67 @@
 
 ![The Bonsai Platform][1]
 
-## Build
+###### Build
 
-* **CLI:** the Bonsai Command Line Interface (CLI) tool.
-* **brains.bons.ai:** The address for Bonsai's web interface; this is the browser based tool that includes your BRAIN Dashboard and BRAIN Details page.
-* **Simulators:** are a training source. They imitate a real world situation in a virtual environment. Simulators are often a complex, interactive environment. They usually have state, which is a representation of the world inside the virtual environment.
+* **Bonsai Command Line Interface (CLI):** is a command line tool that enables you to configure and control the Bonsai Artificial Intelligence (AI) Engine.
+* **beta.bons.ai:** The address for Bonsai's beta web interface; this is the browser based tool that includes your *BRAIN Dashboard* and *BRAIN Details* page.
+* **Simulators:** are currently the only training source implemented. Simulators are often a complex, interactive virtual environment. They usually have state, which is a representation of the world inside the virtual environment.
 * **Generators:** are a training source. They produce labeled data programmatically. This data is effectively infinite. A generator could, for example, produce a random (but known) integer, set of line segments, etc.
-* **Data:** is a training source. It is information related to the concepts being trained that are used for training or test and evaluate how well the training is implemented. Non-generated data may sometimes be difficult to find, and the data that exists may be a very small set or difficult to label. An example of a data set you could train with is the MNIST database, which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
+* **Data:** is also a training source. It is information related to the scenario being trained, or to test and evaluate how well the training is implemented. Non-generated data may sometimes be difficult to find, and the data that exists may be a very small set or difficult to label. An example of a data set you could train with is the [MNIST database][5], which is a collection of handwritten digits used for training image processing systems and other machine learning systems.
+
+**Note:** Currently, during our private beta, you can only use simulators as your training source.
 
-**Note:** Currently, during our private beta, you can only train with simulators as your training source.
+[//]: # (Simulators are the only training source, beta.bons.ai is current website instead of brains.bons.ai)
 
-## Teach
+###### Teach
 
 * **Inkling:** our unique, special purpose, pedagogical programming language. Pedagogical programming is a new programming paradigm. It expresses the solution to a problem in terms of how you teach the computer to find the solution as opposed to how to calculate the solution.
```

Comment:
I don't think we're trying to teach people what words mean here, if someone (or most people) don't know they can look it up so it doesn't seem like we're crazy people making up words.